app/views/reports/regions/_new_registrations_table.html.erb

```diff
@@ -24,6 +24,7 @@
         <th class="row-label"><strong>Total</strong></th>
         <th class="row-label" colspan="6"><strong>New registrations</strong></th>
       </tr>
+      <% raise "used?" %>
```

Comment:
Pretty sure this view is unused...but wanted to verify with folks before removing.  Couldn't find it in click testing or via this raise.

Reply:
Looks unused to me too 🤷